css/createpoll.css

```diff
@@ -0,0 +1,92 @@
+/* line 1, /www/htdocs/v036898/sites/yagst.de/nc13/server/apps/polls/css/createpoll.scss */
```

Comment:
We should remove this lines everywhere 😛

Reply:
Arrrgh. I will try a local preprocessor.

Reply:
Ok :)